src/Ironclad/Sdk/AuthCookieMiddleware.cs

```diff
@@ -56,5 +66,34 @@ await handler.HandleRequestAsync())
 
             await this.next.Invoke(context);
         }
+
+        private bool UserUsesLatestIos(HttpContext context)
+        {
+            var userAgent = context.Request.Headers["User-Agent"].ToString();
+            var groups = regex.Matches(userAgent);
+
+            if (groups.Any())
```

Comment:
We should probably use the underlying data structure to determine if there are any values eg. `groups.Length` (not sure what the underlying structure is...). Again, for performance reasons.

Reply:
This should also be shortcut to avoid nesting
```suggestion
            if (!groups.Any())
            {
                return false;
            }
```

Reply:
done